layouts/shortcodes/sign-language-section.html

```diff
@@ -0,0 +1,22 @@
+{{/*
+
+  Usage: {{% sign-section VIDEO_ID %}}
+
+  */ -}}
+
+{{ $youtube := "" -}}
+{{ $videoID := .Get 0 -}}
+{{ if $videoID -}}
+  {{ $youtube = printf "{{< youtube id=%q >}}" (printf "%s?mute=1" $videoID) }}
```

Comment:
@chalin I intentionally configured youtube video muted. Is this code acceptable to you ? :) 
The video doesn’t require sound, and some of the current videos have background noise.

Reply:
I think that it would be **preferable for the audio to be stripped** from the videos.

That being said, if you want to enable the [mute](https://gohugo.io/shortcodes/youtube/#mute) argument, it should be done like this:

```suggestion
  {{ $youtube = printf "{{< youtube id=%q mute=true >}}" $videoID }}
```

Reply:
Also, you can use `%s` instead of `%q` here.